devops/azure-pipelines.yml

```diff
@@ -7,7 +7,7 @@ resources:
     type: github
     endpoint: GitHubEmbraceSBSKeycloak
     name: embracesbs/keycloak-containers
-    ref: refs/tags/10.0.3
+    ref: release-embrace
```

Comment:
is this a correct ref value? i thought it should be the git ref, for example refs/heads/master (for master branch)
also, do we want to have a branch as reference instead of a tag, because the content (code) of a tag is static, whilst a branch can get changes over time

Reply:
good point, and agree 100%. I just wanted to validate (run once in a pipeline) this new build script which is now contained in release-embrace branch. My suggestion would be to create a TAG once all new image build process and spi extensions build is validated and we are confident we can promote the commit as release (tag it with new release number)

Reply:
corrected setting resource ref